frodo.js

```diff
@@ -1,66 +1,63 @@
-const lastElement = array[array.length - 1];
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.